### ISSUE TITLE
Add examples of Type I/II TSRK methods

### ISSUE THREAD

**Issue by ketch** [ketch/nodepy#15]
There are functions written specifically for these methods, but no examples or tests using them.

**Comment by ketch** [comment]
Some methods are available in this paper: http://www.davidketcheson.info/assets/papers/2011_tsrk.pdf
See Appendix A.

**Comment by sconde** [comment]
The methods given in the appendix are coefficient of the low storage form. The current status of the TSRK class doesn't have any methods to handle these coefficients. I have an advection example that seems to work but I'm using definition (4.3) in the paper. Is this ok?

**Comment by ketch** [comment]
@sconde You're right.  That low-storage form is, of course, just the canonical Shu-Osher form for TSRKs.  I think the right thing to do is implement a `shu_osher_to_butcher` function for TSRKs (it will be similar to the one for RKs).  I believe this requires inverting the equations that appear below (4.3) in the paper.

**Comment by sconde** [comment]
@ketch, perhaps I'm confused but it seems that a __step__ method is needed. Should I write this for tsrk?

**Comment by ketch** [comment]
You are absolutely right that no `step` method has been implemented for TSRKs.  That would be a very nice contribution!

Of course, examples of TSRKs can be added whether or not we have a step method.

**Comment by sconde** [comment]
@ketch It wasn't my intention to delete any code. I was trying out an idea that may have accidentally been pushed to my master branch rather than my "develop" branch. I shall push the example and my attempt at the __step__ method once I've completed them.

**Comment by ketch** [comment]
Yeah, that's what I thought.  It's no problem; I just wanted to point it out because I wasn't sure you were aware.  I'm looking forward to the new PR!

**Comment by sconde** [comment]
@ketch I'll like to check that my new addition is working, with both fixed-timestep and adpative-time step. Which test would you recommend I run before a PR?

**Comment by ketch** [comment]
Try running a convergence test and a "performance test" with them on any of the nonlinear problems.  See convergence.py and ivp.py.
